src/autopas/iterators/ParticleIterator.h

```diff
@@ -43,6 +47,14 @@ class ParticleIterator : public ParticleIteratorInterfaceImpl<Particle> {
         _iteratorWithinOneCell(cont->begin()->begin()),
         _flagManager(flagManager),
         _behavior(behavior) {
+    size_t myThreadId = autopas_get_thread_num();
+    if (myThreadId < cont->size()) {
+      _iteratorAcrossCells += myThreadId;
+      _iteratorWithinOneCell = _iteratorAcrossCells->begin();
+    } else {
+      _iteratorAcrossCells = cont->end();
```

Comment:
should we eventually spawn a warning in this case, saying that openmp parallelization is inefficient as there are too few cells?

Reply:
added warning
50faeba